src/buildercore/cloudformation.py

```diff
@@ -133,6 +133,12 @@ def read_template(stackname):
     output_fname = os.path.join(config.STACK_DIR, stackname + ".json")
     return json.load(open(output_fname, 'r'))
 
+def read_output(stackname, key):
+    data = core.describe_stack(stackname).meta.data # boto3
+    selected_outputs = [o for o in data['Outputs'] if o['OutputKey'] == key]
```

Comment:
could raise a `ValueError` if these keys do not exist. Ok if this is desired but thought I would check.

Reply:
Yes, it's a low layer in the application so better make it robust